### PR TITLE
Fix cache size and shrinking parameters in onedal iface

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

**Review by michael-smirnov — COMMENTED** [review]
@PetrovKP Don't you think that some tests on shrinking and cache_size have to be added as well?

**Comment by PetrovKP** [comment]
@michael-smirnov both parameters do not functionally affect the algorithm, only the performance

**Comment by michael-smirnov** [comment]
@PetrovKP performance is critical for us. I think it is worth to think how to test that all parameter values are correctly passed to oneDAL C++ API

File: onedal/svm/backend/svm_py.cpp [hunk 1]
```diff
@@ -39,11 +39,11 @@ KernelDescriptor get_kernel_params(const svm_params &params) {
 }
 
 template <typename Result, typename Descriptor, typename... Args>
-Result compute_descriptor_impl(Descriptor descriptor, const svm_params &params, Args &&...args) {
+Result compute_descriptor_impl(Descriptor descriptor, const svm_params &params, Args &&... args) {
     using Task = typename Result::task_t;
     descriptor.set_accuracy_threshold(params.accuracy_threshold)
         .set_max_iteration_count(params.max_iteration_count)
-        .set_cache_size(params.shrinking)
+        .set_cache_size(params.cache_size)
```
Comment (SmirnovEgorRu) [inline onedal/svm/backend/svm_py.cpp:46]: :D

**Review by michael-smirnov — APPROVED** [review]

**Pull request merged**